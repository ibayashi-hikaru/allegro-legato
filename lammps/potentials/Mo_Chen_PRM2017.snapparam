# DATE: 2017-09-18 UNITS: metal CONTRIBUTOR: Chi Chen <chc273@eng.ucsd.edu> CITATION: C. Chen, Z. Deng, R. Tran, H. Tang, I.-H. Chu, S. P. Ong, "Accurate force field for molybdenum by machine learning large materials data" Physical Review Materials 1, 04 3603 (2017)
# Generated by Materials Virtual Lab
rcutfac 4.615858
twojmax 6
