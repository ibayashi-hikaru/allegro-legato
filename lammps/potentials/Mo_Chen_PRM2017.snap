
# DATE: 2017-09-18 UNITS: metal CONTRIBUTOR: Chi Chen <chc273@eng.ucsd.edu> CITATION: C. Chen, Z. Deng, R. Tran, H. Tang, I.-H. Chu, S. P. Ong, "Accurate force field for molybdenum by machine learning large materials data" Physical Review Materials 1, 04 3603 (2017)
# Generated by Materials Virtual Lab
# Definition of SNAP potential.
pair_style snap
pair_coeff * * Mo_Chen_PRM2017.snapcoeff Mo_Chen_PRM2017.snapparam Mo

